pages/__dashboard.tsx

```diff
@@ -0,0 +1,28 @@
+import Seo from 'components/Layout/Seo/Seo';
+import MyAllocatedCases from 'components/AllocatedCases/MyAllocatedCases';
+import DashboardWrapper from 'components/Dashboard/DashboardWrapper';
+
+/**
+ * This page, the Dashboard, is being exposed for internal testing at `{domain}/__dashboard`.
+ *
+ * Once the Dashboard is ready to launch, this file should be removed, and the redirect
+ *  (see `getServerSideProps()` in `/pages/index.tsx`) should be removed, so the
+ *  dashboard is available at the `{domain}/` route.
+ *
+ * `/components/Dashboard/DashboardWrapper.tsx` will also need to change to modify the base path of
+ *  the dashboard from `/__dashboard` to `/`.
```

Comment:
Just checking my understanding, after testing and this has been given the ✅ from stakeholders, the dashboard would be the new home/landing page instead of the search. 🙂

Reply:
That's absolutely right! 🎉